sdk/nodejs/helm/v2/helm.ts

```diff
@@ -345,14 +340,13 @@ interface ResolvedFetchOpts {
  */
 export function fetch(chart: string, opts?: ResolvedFetchOpts) {
     const flags: string[] = [];
+    const env: { [key: string]: string | undefined } = {...process.env};
     if (opts !== undefined) {
         // Untar by default.
         if(opts.untar !== false) { flags.push(`--untar`); }
 
-        // Fallback to using the HELM_HOME environment variable if opts.home is not set.
-        if (!opts.home) {
-            opts.home = process.env.HELM_HOME || undefined;
-        }
+        // Helm v3 removed the `--home` flag, so we must use an env var instead.
```

Comment:
Is this new way okay for v2 as well?

Reply:
Yep, `HELM_HOME` works on both versions.